judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueriesWithDefaultValues.java

```diff
@@ -0,0 +1,205 @@
+                .withFldBool(true)
+                .withFldDate(LocalDate.parse("1900-01-01"))
+                .withFldTime(LocalTime.parse("12:00:13"))
+                .withFldTimestamp(OffsetDateTime.parse("2020-01-01T01:11:13-12:00").atZoneSameInstant(ZoneOffset.UTC).toLocalDateTime())
+                .withFldLong(1234567890L)
+                .withFldString("AAA")
+                .withFldDecimal(-1.2302)
+                .withFldEnum(MyEnum.A03)
+                .build());
+
+        MyEntity e3 = myEntityDao.create(MyEntity.builder()
+                .withFldBool(false)
+                .withFldDate(LocalDate.now())
+                .withFldTime(LocalTime.parse("15:15:01"))
+                .withFldTimestamp(OffsetDateTime.parse("2020-01-01T01:11:13+01:00").atZoneSameInstant(ZoneOffset.UTC).toLocalDateTime())
+                .withFldLong(12345678999L)
+                .withFldString("AAA")
+                .withFldDecimal(-2.2302)
+                .withFldEnum(MyEnum.A02)
+                .build());
+
+        MyEntity e4 = myEntityDao.create(MyEntity.builder()
+                .withFldBool(true)
+                .withFldDate(LocalDate.parse("2023-01-01"))
+                .withFldTime(LocalTime.parse("15:20"))
+                .withFldTimestamp(OffsetDateTime.parse("2020-01-01T01:11:13+01:00").atZoneSameInstant(ZoneOffset.UTC).toLocalDateTime())
+                .withFldLong(-16L)
+                .withFldString("Lorem ipsum")
+                .withFldDecimal(999999999.9999)
+                .withFldEnum(MyEnum.A01)
+                .build());
+
+        
+        Snapshot1 s1 = snapshot1Dao.create(Snapshot1.builder().build());
+        assertEquals(e2.identifier().getIdentifier(), snapshot1Dao.queryFldMyEntity001(s1).orElseThrow().identifier().getIdentifier());
+        assertEquals(e3.identifier().getIdentifier(), snapshot1Dao.queryFldMyEntity002(s1).orElseThrow().identifier().getIdentifier());
+        assertTrue(
+                    snapshot1Dao.queryFldMyEntity003(s1).orElseThrow().identifier().getIdentifier().equals(e3.identifier().getIdentifier())
+                    || snapshot1Dao.queryFldMyEntity003(s1).orElseThrow().identifier().getIdentifier().equals(e4.identifier().getIdentifier())
+                );
+        assertEquals(e3.identifier().getIdentifier(), snapshot1Dao.queryFldMyEntity004(s1).orElseThrow().identifier().getIdentifier());
+        assertEquals(e4.identifier().getIdentifier(), snapshot1Dao.queryFldMyEntity005(s1).orElseThrow().identifier().getIdentifier());
+        assertTrue(
+                snapshot1Dao.queryFldMyEntity006(s1).orElseThrow().identifier().getIdentifier().equals(e2.identifier().getIdentifier())
+                || snapshot1Dao.queryFldMyEntity006(s1).orElseThrow().identifier().getIdentifier().equals(e3.identifier().getIdentifier())
+                || snapshot1Dao.queryFldMyEntity006(s1).orElseThrow().identifier().getIdentifier().equals(e4.identifier().getIdentifier())
+            );
```

Comment:
Considering that the results of `snapshot1Dao.queryFldMyEntity003` and `snapshot1Dao.queryFldMyEntity006` can differ on each run, please extract their results into local variables to be able to compare the expected values with only "one result".